packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -1698,7 +1797,7 @@ describe('NetworkController', () => {
                     'AAAA-AAAA-AAAA-AAAA',
                     {
                       type: NetworkClientType.Custom,
-                      chainId: toHex(1),
+                      chainId: '0x1',
```

Comment:
Should revert this change

```suggestion
                      chainId: toHex(1),
```